src/containers/ConversionForm/index.tsx

```diff
@@ -70,21 +71,28 @@ const ConversionForm = ({
     receiveFileToConvert,
     initializeConversion,
     conversionStatus,
+    convertFile,
+    setConversionStatus,
 }: ConversionProps): JSX.Element => {
     const [fileToConvert, setFileToConvert] = useState<UploadFile | null>();
     const [engineSelected, setEngineSelected] = useState<boolean>(false);
+    const [isProcessing, setIsProcessing] = useState<boolean>(false);
     const [serverDownModalOpen, setServerIsDownModalOpen] =
```

Comment:
nit (and not even one in the diff!): inconsistent names: `serverDownModalOpen` vs `setServer*Is*DownModalOpen`

Reply:
Good catch, I think I now prefer serverErrorModalOpen anyway as it matches the modal component better